base_location/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,13 @@
+#. Activate the developer mode in *Settings*.
+#. Go to *Settings / Technical / Locations Management / Locations*.
```

Comment:
This menu no longer exists on v12, and I think that we should put Locations menu now on *Contacts > Configuration > Localization*, as we have that menu path for the rest of the localization elements (States, Countries...), and thus to add `contacts` as dependency, which is very little trade-off.